[20160929-083450]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirCampanhaEntrada.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-083450]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-083450]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-083450]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-083450]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-083450]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-083450]: 	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
[20160929-083450]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:624)
[20160929-083450]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-083450]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-083450]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-083450]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-083450]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-083450]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-083450]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-083450]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-083450]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-083450]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-083450]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-083450]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-083450]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-083450]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-083450]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-083450]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-083450]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-083450]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-083450]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-083450]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-083450]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-083450]: Caused by: java.lang.NullPointerException
[20160929-083450]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:95)
[20160929-083450]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-083450]: 	... 26 more
[20160929-083450]: .
[20160929-084355]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-084355]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-084355]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-084355]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-084355]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-084355]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-084355]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-084355]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-084355]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-084355]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-084355]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-084355]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-084355]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-084355]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-084355]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-084355]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-084355]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-084355]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-084355]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-084355]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-084355]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-084355]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-084355]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-084355]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-084355]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-084355]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-084355]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-084355]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-084355]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-084355]: Caused by: java.lang.NullPointerException
[20160929-084355]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-084355]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-084355]: 	... 26 more
[20160929-084355]: .
[20160929-084928]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-084928]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-084928]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-084928]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-084928]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-084928]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-084928]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-084928]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-084928]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-084928]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-084928]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-084928]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-084928]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-084928]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-084928]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-084928]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-084928]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-084928]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-084928]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-084928]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-084928]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-084928]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-084928]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-084928]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-084928]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-084928]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-084928]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-084928]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-084928]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-084928]: Caused by: java.lang.NullPointerException
[20160929-084928]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-084928]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-084928]: 	... 26 more
[20160929-084928]: .
[20160929-094401]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-094401]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-094401]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-094401]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-094401]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-094401]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-094401]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-094401]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-094401]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-094401]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-094401]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-094401]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-094401]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-094401]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-094401]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-094401]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-094401]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-094401]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-094401]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-094401]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-094401]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-094401]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-094401]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-094401]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-094401]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-094401]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-094401]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-094401]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-094401]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-094401]: Caused by: java.lang.NullPointerException
[20160929-094401]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-094401]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-094401]: 	... 26 more
[20160929-094401]: .
[20160929-094403]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-094403]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-094403]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-094403]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-094403]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-094403]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-094403]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-094403]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-094403]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-094403]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-094403]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-094403]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-094403]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-094403]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-094403]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-094403]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-094403]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-094403]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-094403]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-094403]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-094403]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-094403]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-094403]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-094403]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-094403]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-094403]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-094403]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-094403]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-094403]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-094403]: Caused by: java.lang.NullPointerException
[20160929-094403]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-094403]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-094403]: 	... 26 more
[20160929-094403]: .
[20160929-094442]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-094442]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-094442]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-094442]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-094442]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-094442]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-094442]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-094442]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-094442]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-094442]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-094442]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-094442]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-094442]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-094442]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-094442]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-094442]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-094442]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-094442]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-094442]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-094442]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-094442]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-094442]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-094442]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-094442]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-094442]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-094442]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-094442]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-094442]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-094442]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-094442]: Caused by: java.lang.NullPointerException
[20160929-094442]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-094442]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-094442]: 	... 26 more
[20160929-094442]: .
[20160929-094519]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-094519]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-094519]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-094519]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-094519]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-094519]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-094519]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-094519]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-094519]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-094519]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-094519]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-094519]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-094519]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-094519]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-094519]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-094519]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-094519]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-094519]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-094519]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-094519]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-094519]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-094519]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-094519]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-094519]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-094519]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-094519]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-094519]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-094519]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-094519]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-094519]: Caused by: java.lang.NullPointerException
[20160929-094519]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-094519]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-094519]: 	... 26 more
[20160929-094519]: .
[20160929-095551]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'null', StackTrace:=java.lang.NullPointerException
[20160929-095551]: 	at org.apache.struts.action.RequestProcessor.getInternal(RequestProcessor.java:1126)
[20160929-095551]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:538)
[20160929-095551]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-095551]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-095551]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-095551]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-095551]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-095551]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-095551]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-095551]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-095551]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-095551]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-095551]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-095551]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-095551]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-095551]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-095551]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-095551]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-095551]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-095551]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-095551]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-095551]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-095551]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-095551]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-095551]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-095551]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-095551]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-095551]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-095551]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-095551]: .
[20160929-102249]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', conteudoImagem.enURLConteudoCampanha='5265', conteudoImagem.enFisicoConteudoCampanha='C:\storage\999\5265\Teste.pdf', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-102249]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-102249]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-102249]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-102249]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-102249]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-102249]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-102249]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-102249]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-102249]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-102249]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-102249]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-102249]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-102249]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-102249]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-102249]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-102249]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-102249]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-102249]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-102249]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-102249]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-102249]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-102249]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-102249]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-102249]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-102249]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-102249]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-102249]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-102249]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-102249]: Caused by: java.lang.NullPointerException
[20160929-102249]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-102249]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-102249]: 	... 26 more
[20160929-102249]: .
[20160929-103816]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', conteudoImagem.enURLConteudoCampanha='5267', conteudoImagem.enFisicoConteudoCampanha='C:\storage\999\5267\Teste.pdf', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-103816]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-103816]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-103816]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-103816]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-103816]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-103816]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-103816]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-103816]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-103816]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-103816]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-103816]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-103816]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-103816]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-103816]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-103816]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-103816]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-103816]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-103816]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-103816]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-103816]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-103816]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-103816]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-103816]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-103816]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-103816]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-103816]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-103816]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-103816]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-103816]: Caused by: java.lang.NullPointerException
[20160929-103816]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-103816]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-103816]: 	... 26 more
[20160929-103816]: .
[20160929-111954]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirCampanhaEntrada.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-111954]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-111954]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-111954]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-111954]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-111954]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-111954]: 	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
[20160929-111954]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:624)
[20160929-111954]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-111954]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-111954]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-111954]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-111954]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-111954]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-111954]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-111954]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-111954]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-111954]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-111954]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-111954]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-111954]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-111954]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-111954]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-111954]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-111954]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-111954]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-111954]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-111954]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-111954]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-111954]: Caused by: java.lang.NullPointerException
[20160929-111954]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:95)
[20160929-111954]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-111954]: 	... 26 more
[20160929-111954]: .
[20160929-112056]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='asas', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-112056]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-112056]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-112056]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-112056]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-112056]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-112056]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-112056]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-112056]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-112056]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-112056]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112056]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-112056]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-112056]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112056]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-112056]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-112056]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-112056]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-112056]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-112056]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-112056]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-112056]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-112056]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-112056]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-112056]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-112056]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-112056]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-112056]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-112056]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-112056]: Caused by: java.lang.NullPointerException
[20160929-112056]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-112056]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-112056]: 	... 26 more
[20160929-112056]: .
[20160929-112239]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='asas', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-112239]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-112239]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-112239]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-112239]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-112239]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-112239]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-112239]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-112239]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-112239]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-112239]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112239]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-112239]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-112239]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112239]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-112239]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-112239]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-112239]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-112239]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-112239]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-112239]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-112239]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-112239]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-112239]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-112239]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-112239]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-112239]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-112239]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-112239]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-112239]: Caused by: java.lang.NullPointerException
[20160929-112239]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-112239]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-112239]: 	... 26 more
[20160929-112239]: .
[20160929-112347]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='asas', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-112347]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-112347]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-112347]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-112347]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-112347]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-112347]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-112347]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-112347]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-112347]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-112347]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112347]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-112347]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-112347]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112347]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-112347]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-112347]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-112347]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-112347]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-112347]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-112347]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-112347]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-112347]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-112347]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-112347]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-112347]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-112347]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-112347]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-112347]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-112347]: Caused by: java.lang.NullPointerException
[20160929-112347]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-112347]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-112347]: 	... 26 more
[20160929-112347]: .
[20160929-112613]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='asas', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-112613]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-112613]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-112613]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-112613]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-112613]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-112613]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-112613]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-112613]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-112613]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-112613]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112613]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-112613]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-112613]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112613]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-112613]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-112613]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-112613]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-112613]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-112613]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-112613]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-112613]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-112613]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-112613]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-112613]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-112613]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-112613]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-112613]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-112613]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-112613]: Caused by: java.lang.NullPointerException
[20160929-112613]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-112613]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-112613]: 	... 26 more
[20160929-112613]: .
[20160929-112724]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='asas', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-112724]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-112724]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-112724]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-112724]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-112724]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-112724]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-112724]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-112724]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-112724]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-112724]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112724]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-112724]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-112724]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-112724]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-112724]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-112724]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-112724]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-112724]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-112724]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-112724]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-112724]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-112724]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-112724]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-112724]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
[20160929-112724]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-112724]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-112724]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-112724]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-112724]: Caused by: java.lang.NullPointerException
[20160929-112724]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-112724]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-112724]: 	... 26 more
[20160929-112724]: .
[20160929-113754]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abc', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-113754]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-113754]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-113754]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-113754]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-113754]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-113754]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-113754]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-113754]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-113754]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-113754]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-113754]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-113754]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-113754]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-113754]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-113754]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-113754]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-113754]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-113754]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-113754]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-113754]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-113754]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-113754]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-113754]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-113754]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-113754]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-113754]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-113754]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-113754]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-113754]: Caused by: java.lang.NullPointerException
[20160929-113754]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-113754]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-113754]: 	... 26 more
[20160929-113754]: .
[20160929-113952]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-113952]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-113952]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-113952]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-113952]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-113952]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-113952]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-113952]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-113952]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-113952]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-113952]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-113952]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-113952]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-113952]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-113952]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-113952]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-113952]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-113952]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-113952]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-113952]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-113952]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-113952]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-113952]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-113952]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-113952]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-113952]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-113952]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-113952]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-113952]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-113952]: Caused by: java.lang.NullPointerException
[20160929-113952]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-113952]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-113952]: 	... 26 more
[20160929-113952]: .
[20160929-114410]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-114410]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-114410]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-114410]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-114410]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-114410]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-114410]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-114410]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-114410]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-114410]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-114410]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-114410]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-114410]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-114410]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-114410]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-114410]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-114410]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-114410]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-114410]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-114410]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-114410]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-114410]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-114410]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-114410]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-114410]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-114410]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-114410]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-114410]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-114410]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-114410]: Caused by: java.lang.NullPointerException
[20160929-114410]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-114410]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-114410]: 	... 26 more
[20160929-114410]: .
[20160929-114900]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-114900]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-114900]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-114900]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-114900]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-114900]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-114900]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-114900]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-114900]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-114900]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-114900]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-114900]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-114900]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-114900]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-114900]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-114900]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-114900]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-114900]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-114900]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-114900]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-114900]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-114900]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-114900]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-114900]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-114900]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-114900]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-114900]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-114900]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-114900]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-114900]: Caused by: java.lang.NullPointerException
[20160929-114900]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-114900]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-114900]: 	... 26 more
[20160929-114900]: .
[20160929-115650]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-115650]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-115650]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-115650]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-115650]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-115650]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-115650]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-115650]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-115650]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-115650]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-115650]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-115650]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-115650]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-115650]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-115650]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-115650]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-115650]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-115650]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-115650]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-115650]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-115650]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-115650]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-115650]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-115650]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-115650]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-115650]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-115650]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-115650]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-115650]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-115650]: Caused by: java.lang.NullPointerException
[20160929-115650]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-115650]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-115650]: 	... 26 more
[20160929-115650]: .
[20160929-115940]: Ocorreu uma exceção no método processActionPerform., CTRL='null', IP='0:0:0:0:0:0:0:1', contextPath='/calipacampanhas', URI='/calipacampanhas/IncluirInformativoConfirmar.do', Mensagem='Ocorreu uma exceção no método processActionPerform.', Parâmetros:, CTRL='null', fakefiletext1='Teste.pdf', campanha.cdCategoriaPagina='2', cdAmbienteInternet='1', conteudoImagem.dsTextoConteudoCampanha='abcd', campanha.nmCampanha='teste123', campanha.categoriaInformativo.cdCategoriaInformativo='1', CTRL='null', Mensagem da Exceção:'java.lang.NullPointerException', StackTrace:=javax.servlet.ServletException: java.lang.NullPointerException
[20160929-115940]: 	at org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:545)
[20160929-115940]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:486)
[20160929-115940]: 	at br.com.bradesco.portaladm.apresentacao.AplicacaoPortalRequestProcessor.processActionPerform(AplicacaoPortalRequestProcessor.java:79)
[20160929-115940]: 	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
[20160929-115940]: 	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
[20160929-115940]: 	at org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:525)
[20160929-115940]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
[20160929-115940]: 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
[20160929-115940]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
[20160929-115940]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-115940]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CharEncodeFilter.doFilter(CharEncodeFilter.java:37)
[20160929-115940]: 	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
[20160929-115940]: 	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
[20160929-115940]: 	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
[20160929-115940]: 	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
[20160929-115940]: 	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
[20160929-115940]: 	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
[20160929-115940]: 	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
[20160929-115940]: 	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:956)
[20160929-115940]: 	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
[20160929-115940]: 	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
[20160929-115940]: 	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
[20160929-115940]: 	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:625)
[20160929-115940]: 	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
[20160929-115940]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
[20160929-115940]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
[20160929-115940]: 	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
[20160929-115940]: 	at java.lang.Thread.run(Thread.java:662)
[20160929-115940]: Caused by: java.lang.NullPointerException
[20160929-115940]: 	at br.com.bradesco.custodia.portal.campanhas.apresentacao.CampanhasBaseAction.execute(CampanhasBaseAction.java:114)
[20160929-115940]: 	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
[20160929-115940]: 	... 26 more
[20160929-115940]: .
